hack/build-docker.sh

```diff
@@ -40,6 +40,7 @@ for arg in $args; do
     if [ "${target}" = "build" ]; then
         (
             cd ${CMD_OUT_DIR}/${BIN_NAME}/
+            git describe > .gitinfo
```

Comment:
Can you change the PR to get the needed info out of `hack/version.sh`? Maybe adding an extra function there. This file is currently used to get such info from git, or alternatively get it from environment variables to allow building outside of the git repo.